package.json

```diff
@@ -407,6 +407,50 @@
                     "additionalProperties": {
                       "type": "string"
                     }
+                  },
+                  "ports": {
+                    "type": "array",
+                    "description": "Ports that are going to be mapped on the host.",
+                    "hostPort": {
```

Comment:
I'm not sure this is [JSON Schema format](https://json-schema.org/).  I thought that it would look something like:

```json
{
  "ports": {
    "type": "array",
    "description": "...",
    "items": {
      "type": "object",
      "properties": {
        "hostPort": { ... },
        "containerPort": { ... },
        "protocol": { ... }
      },
      "required": [ "containerPort" ]
    }
  }
}
```

Reply:
Sorry, the objects were malformed, I'll also apply enums for the protocol and permissions properties so it shows the user by intelliSense which elements to insert on those fields.